modules/nw-network-flows-create.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/tracking-network-flows.adoc
+
+[id="nw-network-flows-create_{context}"]
+= Adding destinations for network flows collectors
+
+As a cluster administator, you can configure the Cluster Network Operator to send network flows metadata about the pod network to a network flows collector.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You are logged in to the cluster with a user with `cluster-admin` privileges.
+* You have a network flows collector and know the IP address and port that it listens on.
+
+.Procedure
+
+. Create a manifest file that specifies the network flows collector type and the IP address and port information of the collectors.
+
+. Run the following command to configure the Cluster Network Operator with the network flows collectors:
```

Comment:
I've preferred the _To do this thing, enter the following command:_ construction because it puts the thing first.

Reply:
Thank you for pointing it out.  I revised again to just "Configure the CNO with the network flows collectors:".  If that is still ick, please say so.

I also tried to add information about the questions you asked in the Jira.  Primarily about whether each destination receives the same data, round-robin, and so on.